.github/copilot/integrate-social-share-button.prompt.md

```
@@ -1,653 +1,261 @@
+Add CDN `<link>` & `<script>` tags to `index.html`. In `onMounted`, instantiate `new window.SocialShareButton({ container: containerRef.value, buttonStyle: "default" })`.
 
-Tell them to copy `src/social-share-button-qwik.tsx` from the library into their project:
+---
 
-```tsx
-import { component$ } from "@builder.io/qwik";
-import { SocialShareButton } from "./components/SocialShareButton";
+### 🅰️ Angular
+
+Component file (`share-button.component.ts`):
+```typescript
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)